pallets/elections/src/lib.rs

```diff
@@ -202,13 +248,87 @@ pub mod pallet {
 
             Ok(())
         }
+
+        /// Sets kick-out config (in effect in the next era)
+        #[pallet::weight((T::BlockWeights::get().max_block, DispatchClass::Operational))]
+        pub fn set_kick_out_config(
+            origin: OriginFor<T>,
+            minimal_expected_performance: Option<u8>,
+            underperformed_session_count_threshold: Option<u32>,
+            clean_session_counter_delay: Option<u32>,
+        ) -> DispatchResult {
+            ensure_root(origin)?;
+
+            let mut current_committee_kick_out_config = NextEraCommitteeKickOutConfig::<T>::get();
+
+            if let Some(minimal_expected_performance) = minimal_expected_performance {
+                ensure!(
+                    minimal_expected_performance <= 100,
+                    Error::<T>::InvalidCommitteeKickOutConfig
+                );
+                current_committee_kick_out_config.minimal_expected_performance =
+                    Perbill::from_percent(minimal_expected_performance as u32);
+            }
+            if let Some(underperformed_session_count_threshold) =
+                underperformed_session_count_threshold
+            {
+                ensure!(
+                    underperformed_session_count_threshold > 0,
+                    Error::<T>::InvalidCommitteeKickOutConfig
+                );
+                current_committee_kick_out_config.underperformed_session_count_threshold =
+                    underperformed_session_count_threshold;
+            }
+            if let Some(clean_session_counter_delay) = clean_session_counter_delay {
+                ensure!(
+                    clean_session_counter_delay > 0,
+                    Error::<T>::InvalidCommitteeKickOutConfig
+                );
+                current_committee_kick_out_config.clean_session_counter_delay =
+                    clean_session_counter_delay;
+            }
+
+            NextEraCommitteeKickOutConfig::<T>::put(current_committee_kick_out_config.clone());
+            Self::deposit_event(Event::SetCommitteeKickOutConfig(
+                current_committee_kick_out_config,
+            ));
+
+            Ok(())
+        }
+
+        /// Remove node from the committee.
```

Comment:
The comment could be more precise, for instance: "Schedule a non-reserved node to be kicked out from the committee at the end of the era."

Reply:
Sure, done. The reason I've kept it concise is that there's a limit of characters next to the extrinsic description in UI, but this time it fits. Otherwise, it cuts the sentence displaying this  `...` which might be annoying to the user.  

![image](https://user-images.githubusercontent.com/3909333/195008565-1e2f0fed-46ee-4943-b39a-aa921326bbf0.png)